### PR TITLE
fishtown-analytics => dbt-labs; master to main where its default

### PR DESCRIPTION
**Pull request by lostmygithubaccount**
## Description & motivation
A) getting in the rhythm of contributing to docs! B) noticed "fishtown-analytics" in the `README.md` and figured I'd try to clean it up throughout. I did not change blogs. Otherwise, I found/replaced "fishtown-analytics" with "dbt-labs", "dbt" with "dbt-core" as the repo name where appropriate, and "master" with "main" for git links where the default branch has changed accordingly. checked that all links still work

## To-do before merge
nothing? hopefully nothing conflicting 

## Prerelease docs
N/A

## Checklist
N/A

If you removed existing pages (delete if not applicable):
N/A

### CODE REVIEW

**Review by joellabes — APPROVED** [review]
Yay! thank you ⭐

File: website/docs/reference/resource-properties/description.md [hunk 1]
```diff
@@ -357,7 +357,7 @@ version: 2
 
 models:
   - name: customers
-    description: "!\[dbt Logo](https://raw.githubusercontent.com/dbt-labs/dbt/develop/etc/dbt-logo.svg)"
+    description: "!\[dbt Logo](https://raw.githubusercontent.com/dbt-labs/dbt-core/develop/etc/dbt-logo.svg)"
```
Comment (runleonarun) [inline website/docs/reference/resource-properties/description.md]: Both of these links 404 for me. The `raw.githubusercontent.com` means it was created from an image in a PR. Do we want to link to this image? https://github.com/dbt-labs/dbt-core/blob/main/etc/dbt-core.svg

**Review by runleonarun — APPROVED** [review]
@lostmygithubaccount This looks great! Just one image question but otherwise 🚢 it!

File: website/docs/reference/resource-properties/description.md (hunk 1 above)
Comment (lostmygithubaccount) [reply website/docs/reference/resource-properties/description.md]: swapped

**Pull request merged**